src/runtime/runtime.js

```diff
@@ -387,6 +397,11 @@ export class ConfiguredRuntime {
     this.callbacks_.setOnEntitlementsResponse(callback);
   }
 
+  /** @override */
+  getOffers(opt_options) {
```

Comment:
duplicate of line 237?

Reply:
No, that's the distinction between pending and configured runtime. The pending runtime is a simple proxy.